### DISCUSSION TITLE
Install Advanced button in ChocolateyGUI

### DISCUSSION THREAD

**Discussion by lottemanuputty** [chocolatey/ChocolateyGUI#1059] [Q&A]
Hi there community,

I was wondering if you know there is a possibility to make the 'Install Advanced' button invisible, so the user who would like to install any kind of application in the store, only can see the 'Install' button?

Please let me know, thank you!

Greetings Lotte

**Comment by gep13** [comment]
@lottemanuputty there is no way, currently, to show/hide this button.

However, there is an [issue](https://github.com/chocolatey/ChocolateyGUI/issues/1055), which is slated to be in the next release of Chocolatey GUI, which would allow control of this button via the Chocolatey GUI Licensed Extension.
